doc/source/changes/version_0_23.rst.inc

```diff
@@ -110,6 +110,11 @@
     >>> wb[0]
     <larray.io.excel.Sheet [Book1]Sheet1>
 
+* path to the directory continaing example files is given the variable `EXAMPLE_FILES_DIR`:
+
+    >>> from larray import *
+    >>> pop = read_csv(EXAMPLE_FILES_DIR + 'pop.csv')
```

Comment:
I wonder if EXAMPLE_DATA_DIR wouldn't be slightly better? (to be consistent with load_example_data)